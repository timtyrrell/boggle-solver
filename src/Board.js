import React, { Component } from 'react'

class Board extends Component {
  render() {
    return (
      <div className="Board">
        render checkbox and textboxes here
      </div>
    )
  }
}

export default Board
