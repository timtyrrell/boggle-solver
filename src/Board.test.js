import React from 'react'
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer';
import Board from './Board'


it('renders without crashing', () => {
  const component = renderer.create(<Board />)
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
