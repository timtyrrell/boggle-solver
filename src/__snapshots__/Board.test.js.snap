// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="Board"
>
  render checkbox and textboxes here
</div>
`;
